config/config.js

```diff
@@ -0,0 +1,242 @@
+require("dotenv").config()
```

Comment:
A note: dependabot just bumped `dotenv` to `10.0.0` in https://github.com/isomerpages/isomercms-backend/pull/219 so we can now use `import` instead of `require`.

https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md